reana/reana_dev/git.py

```diff
@@ -329,6 +329,56 @@ def git_fork(component, exclude_components, browser):  # noqa: D301
     )
 
 
+@click.option(
+    "--component",
+    "-c",
+    multiple=True,
+    default=["CLUSTER"],
+    help="Which components? [shortname|name|.|CLUSTER|ALL]",
+)
+@click.option(
+    "--exclude-components",
+    default="",
+    help="Which components to exclude? [c1,c2,c3]",
+)
+@git_commands.command(name="git-fork-automatic")
+def git_fork_automatic(component, exclude_components):  # noqa: D301
+    """Fork REANA source code repositories.
+
+    \b
+    :param components: The option ``component`` can be repeated. The value may
+                       consist of:
+                         * (1) standard component name such as
+                               'reana-workflow-controller';
+                         * (2) short component name such as 'r-w-controller';
+                         * (3) special value '.' indicating component of the
+                               current working directory;
+                         * (4) special value 'CLUSTER' that will expand to
+                               cover all REANA cluster components [default];
+                         * (5) special value 'CLIENT' that will expand to
+                               cover all REANA client components;
+                         * (6) special value 'DEMO' that will expand
+                               to include several runable REANA demo examples;
+                         * (7) special value 'ALL' that will expand to include
+                               all REANA repositories.
+    :param exclude_components: List of components to exclude.
+    :type component: List[str]
+    :type exclude_components: str
+    """
+    if exclude_components:
+        exclude_components = exclude_components.split(",")
+    components = select_components(component, exclude_components)
+    for component in components:
+        # Use the gh CLI to fork the repository
+        cmd = f"gh repo fork reanahub/{component} --clone=false"
+        run_command(cmd)
+        msg = f"Repository {component} forked and cloned successfully."
+        display_message(msg)
+    display_message(
+        "All repositories forked successfully. To clone them locally run: reana-dev git-clone"
```

Comment:
The given command is not complete, so I would either:

- say something generic like _You can clone them locally with `git-clone`_
- show the complete command that can be copy-pasted and executed directly, which should include the username but also all the `-c` and `--exclude-components` options that were passed to git-fork-automatic

What do you think?

Reply:
Since we don't have the user anymore in this command, I wouldn't list the copy-pastable command (that would have a placeholder for the username), so I added a note in the description of the command, and changed the message to the first of the two options you suggested.